{
    "base-errors-activity_invite_already_invited": "De upper __name__ is al uitgenodigd voor deze activiteit"
}
